.github/workflows/sdk-breaking-change-labels.yaml

```diff
@@ -0,0 +1,55 @@
+name: SDK Breaking Change Labels (Preview)
+
+on:
+  check_run:
```

Comment:
Do you need to trigger on check_run:completed, or would check_suite:completed be sufficient?

Reply:
As there are a few check runs in the 'Azure Pipeline' suite, I don't want the triggering needs to wait until all the check runs complete.